controllers/socialevent/socialevent_controller_test.go

```diff
@@ -0,0 +1,120 @@
+package socialevent_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/controllers/socialevent"
+	"github.com/codeready-toolchain/host-operator/pkg/apis"
+	tiertest "github.com/codeready-toolchain/host-operator/test/nstemplatetier"
+	socialeventtest "github.com/codeready-toolchain/host-operator/test/socialevent"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestReconcileSocialEvent(t *testing.T) {
+
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
+	err := apis.AddToScheme(scheme.Scheme)
+	require.NoError(t, err)
+	basicTier := tiertest.BasicTier(t, tiertest.CurrentBasicTemplates)
+
+	t.Run("valid tier", func(t *testing.T) {
+		// given
+		se := socialeventtest.NewSocialEvent("lab", "basic")
+		hostClient := test.NewFakeClient(t, se, basicTier)
+		ctrl := newReconciler(hostClient)
+
+		// when
+		res, err := ctrl.Reconcile(context.TODO(), requestFor(se))
+
+		// then
+		require.NoError(t, err)
+		assert.False(t, res.Requeue)
```

Comment:
the `Requeue` should be false in all cases if I'm not mistaken. We don't verify that in such cases, so you could skip it here as well.

Reply:
it's not a big deal, is it?

Reply:
removed in c0aefb2 and 026e91c